vscode/src/testController.ts

```diff
@@ -580,8 +580,8 @@ export class TestController {
     // test framework is loaded, it might change which options are accepted. For example, if we append `-r` after the
     // file path for Minitest, it will fail with unrecognized argument errors
     const rubyOpt = workspace.ruby.env.RUBYOPT
-      ? `${workspace.ruby.env.RUBYOPT} ${response.reporterPaths?.map((path) => `-r${path}`).join(" ")}`
-      : response.reporterPaths?.map((path) => `-r${path}`).join(" ");
+      ? `${workspace.ruby.env.RUBYOPT} -rbundler/setup ${response.reporterPaths?.map((path) => `-r${path}`).join(" ")}`
+      : `-rbundler/setup ${response.reporterPaths?.map((path) => `-r${path}`).join(" ")}`;
```

Comment:
Minor nit, but could we extract out the common parts here? It's hard to know what's actually different

```suggestion
    const commonOpts = `-rbundler/setup ${response.reporterPaths?.map((path) => `-r${path}`).join(" ")}`
    
    const rubyOpt = workspace.ruby.env.RUBYOPT ? `${workspace.ruby.env.RUBYOPT} ${commonOpts}` : commonOpts;
```